src/components/PresentBar/PresentBar.tsx

```diff
@@ -14,26 +14,76 @@
  * limitations under the License.
  */
 
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
 import PresentToAllIcon from '@mui/icons-material/PresentToAll';
 import { Box } from '@mui/material';
+import { findIndex } from 'lodash';
+import { useCallback } from 'react';
 import { useTranslation } from 'react-i18next';
-import { PresentationState, usePresentationMode } from '../../state';
+import {
+  PresentationState,
+  useActiveSlide,
+  useActiveWhiteboardInstance,
+  useActiveWhiteboardInstanceSlideIds,
+  usePresentationMode,
+} from '../../state';
 import { useUserDetails } from '../../store';
-import { Toolbar, ToolbarAvatar, ToolbarToggle } from '../common/Toolbar';
+import {
+  Toolbar,
+  ToolbarAvatar,
+  ToolbarButton,
+  ToolbarToggle,
+} from '../common/Toolbar';
 
 export function PresentBar() {
   const { t } = useTranslation();
-
+  const slideIds = useActiveWhiteboardInstanceSlideIds();
+  const whiteboardInstance = useActiveWhiteboardInstance();
+  const { activeSlideId, isFirstSlideActive, isLastSlideActive } =
+    useActiveSlide();
   const { state, togglePresentation } = usePresentationMode();
 
+  const isViewingPresentation = state.type === 'presenting';
+
+  const handleToNextSlideClick = useCallback(() => {
+    if (activeSlideId) {
+      const activeSlideIndex = findIndex(
+        slideIds,
+        (value) => value === activeSlideId
+      );
+      whiteboardInstance.setActiveSlideId(slideIds[activeSlideIndex + 1]);
+    }
```

Comment:
You can also use `indexOf` instead of lodash.

```suggestion
    if (activeSlideId) {
      const activeSlideIndex = slideIds.indexOf(activeSlideId);
      whiteboardInstance.setActiveSlideId(slideIds[activeSlideIndex + 1]);
    }
```